### ISSUE TITLE
"Payment Date" displayed on unpaid invoices with pending payment status

### ISSUE THREAD

**Issue by luke-au** [restrictcontentpro/restrict-content-pro#1668]
Invoices for manual payments with a pending status still display "Payment Date" under the Additional Info section of their invoices. 
Should this instead be conditionally displayed depending on the invoices payment status? 
Perhaps using a hook for the above invoice notes section.

**Comment by nosegraze** [comment]
Yes I definitely think we could only show that if the payment status is `complete`. If not, we could either omit it entirely or simply adjust the text to say `Due Date` instead.

**Comment by tnorthcutt** [comment]
For `Complete` and `Refunded` statuses, it probably makes sense to show the payment date.

For `Pending`, `Failed`, and `Abandoned`, we should probably hide the payment date field.
